velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -25,76 +25,85 @@ namespace {
 static constexpr char kWomanFacepalmingLightSkinTone[] =
     "\xF0\x9F\xA4\xA6\xF0\x9F\x8F\xBB\xE2\x80\x8D\xE2\x99\x80\xEF\xB8\x8F";
 
-class StringTest : public SparkFunctionBaseTest {
+class StringTest : public test::FunctionBaseTest {
```

Comment:
I'd expect SparkFunctionBaseTest to register only Spark functions. Should we fix it to not register a mix of Spark and Presto functions?

Reply:
Thanks for the advice. Changed FunctionBaseTest to a shared base class without function registration. Do you think it makes sense?